units/crons_remote_minute.go

```diff
@@ -90,6 +90,13 @@ func (j *cronsRemoteMinuteJob) Run(ctx context.Context) {
 		catcher.Add(PopulateCommitQueueJobs(j.env)(ctx, commitQueueQueue))
 	}
 
+	eventNotifierQueue, err := j.env.RemoteQueueGroup().Get(appCtx, "service.event.notifier")
```

Comment:
Can the comment on line 77 extend to this?

Reply:
Done.